unitex/build.xml

```diff
@@ -63,7 +63,9 @@
 
 	<target name="dist" depends="clean,compile" description="generate the distribution">
 		<!-- create Unitex.jar -->
-		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**" />
+		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
+			<zipgroupfileset dir="${lib-dir}" includes="juniversalchardet-1.0.3.jar"></zipgroupfileset>	
```

Comment:
FYI, a maintained version of this library (v2.0.0) is available at https://github.com/albfernandez/juniversalchardet